main.go

```diff
@@ -67,6 +70,7 @@ func printControllerMetaInfo() {
 	setupLog.Info(fmt.Sprintf("go version: %s", runtime.Version()))
 	setupLog.Info(fmt.Sprintf("go os/arch: %s/%s", runtime.GOOS, runtime.GOARCH))
 	setupLog.Info("base logger", "log level", logLevel, "log mode", logMode)
+	setupLog.Info("booting up limitador-operator", "version", version, "commit", commit, "dirty", dirty)
```

Comment:
no need to specify `limitador-operator`. The log belongs to the limitador-operator.

Reply:
Absolutely, will implement!